lib/validators/actions.js

```diff
@@ -147,8 +147,8 @@ export function runKilnjsValidators(schemas = {}, components = {}) {
   Object.keys(components).forEach((uri) => {
     const componentName = getComponentName(uri);
 
-    if (schemas[componentName]) {
-      schemas[componentName].validation.forEach((validationRule) => {
+    if (_.get(schemas, `${componentName}.validation`)) {
```

Comment:
We want to check if the component is there first, I think this doesn't need to change

Reply:
You are right.